pkg/asset/installconfig/platformprovisioncheck.go

```diff
@@ -8,6 +8,7 @@ import (
 	azconfig "github.com/openshift/installer/pkg/asset/installconfig/azure"
 	bmconfig "github.com/openshift/installer/pkg/asset/installconfig/baremetal"
 	gcpconfig "github.com/openshift/installer/pkg/asset/installconfig/gcp"
+	ickubevirt "github.com/openshift/installer/pkg/asset/installconfig/kubevirt"
```

Comment:
Please keep the naming convention like other platforms   (kubevirtconfig / kvconfig)

Reply:
done